resources/assets/themes/base.scss

```diff
@@ -372,3 +372,14 @@ code {
   max-width: 75%;
   display: inline-block;
 }
+
+/* Chrome, Safari, Edge, Opera */
```

Comment:
Can you add a quick note about what this is doing?

Reply:
added